src/main/commandServer/httpCommandServer.ts

```diff
@@ -123,10 +123,19 @@ export class HttpCommandServer {
   }
 
   protected lookupCommand(version: string, method: string, commandName: string) {
-    return this.dispatchTable[version]?.[method]?.[commandName];
+    if (commandName) {
+      return this.dispatchTable[version]?.[method]?.[commandName];
+    }
+    if (version === '' || version in this.dispatchTable) {
+      return (request: http.IncomingMessage, response: http.ServerResponse): Promise<void> => {
+        return this.listEndpoints(version, request, response);
```

Comment:
:cake:
Is this the same as `return this.listEndpoints.bind(this, version)`?
Though, maybe writing it out explicitly the way you have it is clearer…

Reply:
No, I hadn't thought of that. Just read up on `mdn bind` and learned it's not for `this` only.